src/commonTest/kotlin/com/charleskorn/kaml/YamlWritingTest.kt

```diff
@@ -119,7 +120,9 @@ class YamlWritingTest : DescribeSpec({
             val output = Yaml.default.encodeToString(Float.serializer(), 45.6f)
 
             it("returns the value serialized in the expected YAML form") {
-                output shouldBe "45.6"
+                // See a bug in Kotlin/Wasm:
+                // https://youtrack.jetbrains.com/issue/KT-68948/
+                output shouldBeIn setOf("45.6", "45.599998474121094")
```

Comment:
TODO: maybe use `expect`/`actual` to provide the value just for Wasm?

Reply:
https://github.com/charleskorn/kaml/pull/567/commits/72cd9bed809baa52493d848161fb6e0c2083a713